src/gd_kotlin.cpp

```diff
@@ -67,7 +67,7 @@ void GDKotlin::init() {
     jni::Jvm::init(args);
     print_line("Starting JVM ...");
     auto project_settings = ProjectSettings::get_singleton();
-    scripts_root = project_settings->globalize_path("res://src/main/kotlin");
+    scripts_root = "res://src/main/kotlin/";
```

Comment:
I don't mind this change, but there are places that `scripts_root` is used that expects a global path.